core/src/test/java/com/linecorp/armeria/internal/common/metric/CaffeineMetricSupportTest.java

```diff
@@ -47,21 +43,19 @@
 
 import io.micrometer.core.instrument.MeterRegistry;
 
+@ExtendWith(MockitoExtension.class)
 public class CaffeineMetricSupportTest {
```

Comment:
`junit5` test classes don't have to be public
```suggestion
class CaffeineMetricSupportTest {
```